source/patterns/@aws-solutions-constructs/aws-s3-sqs/test/integ.deployQueue.ts

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
What is the purpose of this Integration test ? Don't think the name `integ.deployQueue.ts` captures it.